modules/get-started/pages/quick-start.adoc

```diff
@@ -226,75 +469,249 @@ include::get-started:partial$install-rpk-homebrew.adoc[]
 --
 ======
 
```

Comment:
The macOS tab includes this note: "This method installs the latest version of rpk, which is supported only with the latest version of Redpanda." Would this be confusing to quickstart users? Wouldn't they always be getting the latest version of RP?